game.py

```diff
@@ -5,35 +5,35 @@
 scissors = 'ножницы'
 paper = "бумага"
 x = "да"
-while x == "да":
+while x == "да" or x == "y":
 
-    name = input("что загадываешь? Камень ножницы или бумагу? ")
+    plrin = input("что загадываешь? Камень ножницы или бумагу? ")
```

Comment:
May You please, make this variable more readable (user-friendly)

Reply:
May you please also give possible input options for user?

Reply:
what about  `player_input` ?

Reply:
Sure! Check it out!